src/event_form/steps/event-form-publish.controller.js

```diff
@@ -46,9 +52,27 @@ function EventFormPublishController(
       });
   }
 
+  function publishLater(isEdit) {
+    var modalInstance = $uibModal.open({
+      templateUrl: 'templates/event-form-publish-modal.html',
+      controller: 'EventFormPublishModalController',
+      controllerAs: 'efpmc',
+      resolve: {
+        eventFormData: function () {
+          return controller.eventFormData;
+        },
+        eventCrud : function () {
```

Comment:
eventCrud is never used in EventFormPublishModalController, so this can be deleted